controllers/clusters/cassandra_controller.go

```diff
@@ -1041,7 +1041,7 @@ func (r *CassandraReconciler) newWatchBackupsJob(cluster *v1beta1.Cassandra) sch
 			return err
 		}
 
-		iBackups, err := r.API.GetClusterBackups(instaclustr.ClustersEndpointV1, cluster.Status.ID)
+		iBackups, err := r.API.GetClusterBackups(instaclustr.GetClusterBackupEndpointV2, cluster.Status.ID, instaclustr.ClusterKindsMap[cluster.Kind])
```

Comment:
Form the URL inside the `GetClusterBackupEndpointV2` method please. Fix in all similar cases

Reply:
done